src/Model.php

```diff
@@ -416,7 +416,7 @@ public function init()
             'fields'  => true,
             'scope'   => UserAction\Generic::SINGLE_RECORD,
             'callback'=> 'save',
-            'ui'      => ['icon'=>'edit', 'button'=>[null, 'icon'=>'edit']],
+            'ui'      => ['icon'=>'edit', 'button'=>[null, 'icon'=>'edit'], 'execButton'=>new \atk4\ui\Button(['Save', 'blue'])],
```

Comment:
can this be seed, not the class ? `execButton=>['Button', 'Save']`

Reply:
Yes only seed or nothing (see https://github.com/atk4/data/pull/490#issuecomment-562621547)